db/migrate/20230919200926_create_comment.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComment < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :user, null: false, foreign_key: { to_table: :users }
+      t.references :post, null: false, foreign_key: { to_table: :posts }
+      t.string :text
```

Comment:
- [ ] It would be nice if you could make the type of the comment `text` field be of type `text` as it will be taking a lengthy comment about a post and the `text` type is best for that instead of the `string` type. Kindly use `t.text :text` 👍🏽